projects/overlays/src/services/popovers.service.ts

```diff
@@ -14,6 +14,8 @@ import { PortalLayersService } from './portal-layers.service';
 
 @Injectable({ providedIn: 'root' })
 export class PopoversService {
+  private readonly popoverRefs: Map<ElementRef<HTMLElement> | Position, Popover<any>> = new Map();
```

Comment:
нужно говорящее название для коллекции - что мы достаем и по какому ключу.

+ не используем any
```suggestion
  private readonly popoverRefByAnchor: Map<ElementRef<HTMLElement> | Position, Popover<unknown>> = new Map();
```

Reply:
Решили в личке оставить